src/main/java/org/jenkinsci/plugins/oic/OicSecurityRealm.java

```diff
@@ -509,6 +513,148 @@ private OidcConfiguration buildOidcConfiguration() {
         return conf;
     }
 
+    private static void filterJwsAlgorithms(OIDCProviderMetadata oidcProviderMetadata) {
+        // Filter Jws Algorithms
+        if (oidcProviderMetadata.getIDTokenJWSAlgs() != null) {
```

Comment:
remove the `null` checks and make the filter handle `null` input?

e.g by handling the `null` in `filterFipsNonCompliantJwsAlgorithm` you can have

`oidcProviderMetadate.setIdTokenJWSAlgs(OicAlgorithmValidatorFIPS140.filterFipsNonCompliantJwsAlgorithm(oidcProviderMetadata.getIDTokenJWSAlgs());`